src/index.ts

```diff
@@ -1,3 +1,18 @@
+// import { registerPlugin } from '@capacitor/core';
+
+// import type { SecureStoragePluginPlugin } from './definitions';
+
+// const SecureStoragePlugin = registerPlugin<SecureStoragePluginPlugin>(
+//   'SecureStoragePlugin',
+//   {
+//     web: () => import('./web').then(m => new m.SecureStoragePluginWeb()),
+//   },
+// );
```

Comment:
Why is this commented out? If we don't need it anymore, it should be removed

Reply:
I will remove that.